File: atcoder/go/abc240/a.go
package main

import (
	"fmt"
)

func main() {
	var a, b int
	fmt.Scan(&a, &b)
	if b-a == 1 || (a == 1 && b == 10) {
		fmt.Println("Yes")
	} else {
		fmt.Println("No")
	}
}

File: atcoder/java/abc240/a.java
import java.util.*;

class Main {
    public static void main(String[] args) {
        try(Scanner sc = new Scanner(System.in)) {
            int a = sc.nextInt();
            int b = sc.nextInt();
            if(b-a == 1 || b-a == 9) {
                System.out.println("Yes");
            } else {
                System.out.println("No");
            }
        }
    }
}